lib/model/model_test.go

```diff
@@ -952,6 +952,9 @@ func TestIgnores(t *testing.T) {
 	m.ServeBackground()
 	defer m.Stop()
 
+	// Let initial setup operations (scan, ...) happen
+	time.Sleep(time.Duration(10) * time.Millisecond)
```

Comment:
Totally agree and I must admit, I don't know what it's waiting for exactly. Originally the error occurred at the second and third arrEqual tests. I suspected a race between setting and getting ignores first (as now `GetIgnores` picks the ignores up from memory not disk), but there should be no problem, as setIgnores should block until the new ignores are picked up in the scan it prompts in the end. The delay was introduces by trial and error. Now I removed it again for testing and it works locally, lets see what jenkins means.